samples/tricore-none-eabi/crosstool.config

```diff
@@ -0,0 +1,55 @@
+CT_CONFIG_VERSION="4"
+CT_EXPERIMENTAL=y
+CT_ARCH_TRICORE=y
+CT_ARCH_FLOAT_SW=y
+CT_TARGET_CFLAGS="-ftls-model=local-exec"
+CT_CC_LANG_CXX=y
+CT_TARGET_VENDOR="none"
+CT_BINUTILS_EXTRA_CONFIG_ARRAY="--disable-warn-rwx-segments"
+
+
+CT_GCC_V_13=y
+CT_GCC_VERSION="13.1.0"
+CT_GCC_SRC_DEVEL=y
+CT_GCC_DEVEL_VCS_git=y
+CT_GCC_DEVEL_VCS="git"
+CT_GCC_DEVEL_URL="https://github.com/EEESlab/tricore-gcc.git"
```

Comment:
Is this on track to be in upstream GCC?

Reply:
There are no plans to upstream to GCC repo